src/w3c/templates/cgbg-sotd.js

```diff
@@ -7,15 +7,14 @@ export default (conf, opts) => {
     ${conf.isPreview
       ? html`
           <details class="annoying-warning" open="">
-            <summary
-              >This is a
-              preview${conf.prUrl && conf.prNumber
+            <title>
```

Comment:
Not quite. You need to find where ReSpec creates the `h1` title of the spec AND find where the title element is added and then either replace it or update it there.